nemo/collections/tts/models/aligner.py

```diff
@@ -230,6 +231,19 @@ def setup_test_data(self, cfg):
         pass
 
     @classmethod
-    def list_available_models(cls):
-        """Empty."""
-        pass
+    def list_available_models(cls) -> 'List[PretrainedModelInfo]':
```

Comment:
nit: do we still need a string literal for the type annotation if you've imported the class at Line 27? According to PEP 484, we are good without the quotes (cf https://peps.python.org/pep-0484/#forward-references). Sorry I am not sure about the code convention of NeMo.

Reply:
Just took a look through some of our code, and it looks like we have a mix. A lot of List ones are in quotes, a lot of Optional ones aren't. Not sure if we want to standardize at some point, but changed for now

Reply:
@ericharper do you have any advice on this? Thanks.